docker/docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
     ports:
       - "29092:29092"
       - "9092:9092"
+    volumes:
+      - broker://var/lib/kafka/data/
```

Comment:
can we remove the double slash?

Reply:
@dexter-mh-lee Thanks for your review. I have updated my PR. Please help to squash the commits when applicable.